pkgs/tools/text/wrap/default.nix

```diff
@@ -13,6 +13,8 @@ buildGoModule rec {
 
   nativeBuildInputs = [ makeWrapper ];
 
+  # package still builds but the vendor isn't reproducible with go > 1.17: nix-build -A $name.go-modules --check
+  # may end up needing to mark the package as broken
```

Comment:
Alright, so I'm the maintainer of this one, I got this notification, but I'm not all that familiar with Go.

The package itself builds successfully, the built binary seems to work, and the Nix store location is the same even with `--check`:
```console
nixpkgs (3e17437) took 5s at 21:05:32
❯ nix-build -A wrap --check 1&>2 | rg '/nix/store'
/nix/store/ws3s5slpcvzmi4j0dibnrjlsykh0ijdj-wrap-0.3.1

nixpkgs (3e17437) [?] took 5s at 21:05:50
❯ nix-build -A wrap --check 1&>2 | rg '/nix/store'
/nix/store/ws3s5slpcvzmi4j0dibnrjlsykh0ijdj-wrap-0.3.1
```

But checking the `go-modules` alone fails entirely:
```console
nixpkgs (3e17437) [?] took 5s at 21:06:00
❯ nix-build -A wrap.go-modules --check
checking outputs of '/nix/store/625w5ijcpj31zd0dpl8sri91s9xs270q-wrap-0.3.1-go-modules.drv'...
unpacking sources
unpacking source archive /nix/store/ypfhj27iwcpslsa7hk6famv1jj3lfn7r-source
source root is source
patching sources
applying patch /nix/store/l2wndd70ahws1fi0z21z3m2g4l9fr8bg-courier-prime-variants.patch
patching file pkg/pdf/fonts.go
configuring
building
go: github.com/flopp/go-findfont@v0.0.0-20201103071330-d960cd9a3075: missing go.sum entry; to add it:
        go mod download github.com/flopp/go-findfont
go: github.com/flopp/go-findfont@v0.0.0-20201103071330-d960cd9a3075: missing go.sum entry; to add it:
        go mod download github.com/flopp/go-findfont
error: builder for '/nix/store/625w5ijcpj31zd0dpl8sri91s9xs270q-wrap-0.3.1-go-modules.drv' failed with exit code 1;
```

So what's the takeaway from that?

Reply:
Technically the package is broken as the vendor isn't reproducible with go > 1.17 but I'm not marking it as broken because the vendor is still cached. 

To fix it upstream needs to update their go.{mod,sum} for go > 1.17 and then that change needs to be applied here.

So for now it's fine but it may become a problem at some point and then we'll either need to apply an upstream fix or mark it as broken.